pom.xml

```diff
@@ -105,7 +105,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>1.22</version>
+            <version>2.1.8</version>
```

Comment:
so 2.1.8 was soaked for 2 weeks now... no new bugs on top of 2.1.7... so I am declaring it the preferred version... there is still some issues that plugins have with the 2.1.7+ versions when there is a plugin that iterates an `ExtensionList` before `InitMilestone.EXTENSIONS_AUGMENTED` and can therefore see partially initialized plugins... but that is a problem with the plugins doing bold things, not the credentials plugin ;-)